src/tools/distillery.ml

```diff
@@ -1,12 +1,28 @@
 open Printf
 
-(* Constant use for the filename containing the reserve project name for
- * templates
- *)
+(* File containing the reserve project names for templates *)
 let reserve_project_name_filename = ".reserve"
 
+(* File containing files which must be ignored while copying a template. *)
+let eliomignore_filename          = ".eliomignore"
```

Comment:
I much doubt that you would want to put another `.ignore` in an `eliom-distillery` template (we are not striving for ultimate generality), but in any case, I am fine with `.eliomreserve` and `.eliomignore`. My primary concern is consistency.